### PR TITLE
Add publication policies

### PR DESCRIPTION
**Pull request by nilsvu**
## Proposed changes

This is a proposal for our publication policies. They are intended to work together with our [SXS policies](https://github.com/sxs-collaboration/WelcomeToSXS/blob/master/SxsPolicies.md). They are essentially a more structured version of the implicit publication policies that were used for SpEC papers for years. I appreciate any feedback.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Comment by nilsvu** [comment]
This is ready for review, following our discussion in the call today.

File: docs/PublicationPolicy.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+To aid reproducibility of your scientific results with SpECTRE, we recommend you
+keep track of the version(s) you used and report this information in your
+publication. We also recommend you supply the YAML input files and, if
+appropriate, any additional C++ code you wrote to compile SpECTRE executables as
+supplemental material to the publication.
+
+## Policies that apply to SXS publications {#sxs_publication_policies}
+
+These policies apply to all „science papers” by members of the SXS collaboration
```
Comment (nilsdeppe) [inline docs/PublicationPolicy.md]: we should probably use the US-style quotations to be consistent

File: docs/PublicationPolicy.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  The core developers will discuss among themselves, following the guidelines
+  listed below, and will get back to you with a list of authors.
+
+## Guidelines to decide who has authorship rights on science papers that use SpECTRE
+
+- A contribution to SpECTRE on the order of ~500 lines of code or more should
+  earn you authorship rights on the first science paper that uses the feature.
+  Major contributions, such as writing a full evolution system, earns you
+  authorship rights on the first three science papers that use the feature.
+  „Using the feature” means the code contributes to the result of the paper,
```
Comment (nilsdeppe) [inline docs/PublicationPolicy.md]: US-style quotes?

**Review by nilsdeppe — DISMISSED** [review]
LGTM, couple small questions

File: docs/PublicationPolicy.md (hunk 1 above)
Comment (nilsvu) [reply docs/PublicationPolicy.md]: fixed 👍

**Review by nilsdeppe — DISMISSED** [review]

File: docs/PublicationPolicy.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+To aid reproducibility of your scientific results with SpECTRE, we recommend you
+keep track of the version(s) you used and report this information in your
+publication. We also recommend you supply the YAML input files and, if
+appropriate, any additional C++ code you wrote to compile SpECTRE executables as
+supplemental material to the publication.
+
+## Policies that apply to SXS publications {#sxs_publication_policies}
+
+These policies apply to all "science papers" by members of the SXS collaboration
+that make use of SpECTRE’s code or data, as defined by the [SXS policies](https://github.com/sxs-collaboration/WelcomeToSXS/blob/master/SxsPolicies.md).
```
Comment (wthrowe) [inline docs/PublicationPolicy.md]: Optional: line break.

**Comment by nilsvu** [comment]
@kidder and @wthrowe could you look over this one?

File: docs/PublicationPolicy.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+
+# Publication policies {#publication_policies}
+
+SpECTRE is an open-source code that is developed and maintained by the
+[Simulating eXtreme Spacetimes (SXS) collaboration](https://black-holes.org). We
+ask all authors of scientific publications that make use of SpECTRE's code or
+data to follow the citation guidelines laid out in the section "\ref citing". An
```
Comment (wthrowe) [inline docs/PublicationPolicy.md]: The `\ref`s here aren't formatting correctly, probably because of the quotes.

**Review by wthrowe — CHANGES_REQUESTED** [review]
No problems with the content, just formatting.

**Comment by nilsvu** [comment]
Fixed @wthrowe. @kidder could you give this look as well please?

**Review by kidder — APPROVED** [review]

**Review by nilsdeppe — APPROVED** [review]

**Review by wthrowe — APPROVED** [review]

**Pull request merged**